apps/web/pages/availability/troubleshoot.tsx

```diff
@@ -30,16 +32,27 @@ const AvailabilityView = ({ user }: { user: User }) => {
     }
   );
 
+  const overrides =
+    data?.dateOverrides.reduce((acc, override) => {
```

Comment:
Date overrides aren't shown in troubleshooter if there are more than 1 availabilities and date override is used in second one. 

This seems to be due to the reason that `getUserAvailability picks the first availability/schedule(because eventTypeId isn't provided)
If a default schedule is set, I think that's used.

<img width="1031" alt="Screenshot 2022-12-15 at 1 15 31 PM" src="https://user-images.githubusercontent.com/1780212/207802062-36afb070-1705-4cc8-a764-54731dd152f6.png">

This seems to be a bug in troubleshooter flow and not specific to Date overrides.
cc @emrysal

Reply:
Sounds like a job for a followup